Following errors occurred in deployment. Please fix it.

[11:08:18.667] Running build in Washington, D.C., USA (East) – iad1
[11:08:18.799] Cloning github.com/abhijivani3001/multi-monkey (Branch: newPage, Commit: c8ab098)
[11:08:19.048] Cloning completed: 248.249ms
[11:08:21.185] Restored build cache from previous deployment (5eGQrRnzfWYeSKviKUjrkBhHNzsC)
[11:08:21.254] Running "vercel build"
[11:08:21.716] Vercel CLI 37.12.0
[11:08:22.359] Installing dependencies...
[11:08:23.467] 
[11:08:23.467] up to date in 886ms
[11:08:23.467] 
[11:08:23.467] 70 packages are looking for funding
[11:08:23.468]   run `npm fund` for details
[11:08:23.480] Running "npm run build"
[11:08:23.617] 
[11:08:23.617] > frontend@0.0.0 build
[11:08:23.617] > tsc && vite build
[11:08:23.618] 
[11:08:28.018] src/pages/ForgotPassword.tsx(34,36): error TS6133: 'email' is declared but its value is never read.
[11:08:28.045] Error: Command "npm run build" exited with 2
[11:08:28.523]

--- a/frontend/src/pages/ForgotPassword.tsx
+++ b/frontend/src/pages/ForgotPassword.tsx
@@ -32,6 +32,7 @@ const ForgotPassword = () => {
 
   // Mock function to simulate sending a recovery email
   const sendRecoveryEmail = async (email: string) => {
+    console.log(email);
     return new Promise((resolve) => {
       setTimeout(() => {
         resolve(true);

